packages/commerce-sdk-react/src/hooks/ShopperProducts/mock/index.ts

```diff
@@ -0,0 +1,362 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+export const product = {
```

Comment:
Rather than having all mocked data in one file or folder, I think it's better if each API (like ShopperProducts) has its own `mock` folder. And if this index.ts file is getting too crowded, we have the option to split it into multiple files too.